src/main/configurations/Translate/Configuration_GeefLijstZaakdocumenten_Lv01.xml

```diff
@@ -58,6 +64,111 @@
+                <HttpSender 
+                    name="GetZgwZaakByIdentificatieSender" 
+                    methodType="GET"
+                    headersParams="Authorization,Accept-Crs"
+                    timeout="${zaakbrug.zgw.zaken-api.timeout}"
+                />
+                <Param name="url" value="${zaakbrug.zgw.zaken-api.root-url}zaken"/>
+                <Param name="identificatie" sessionKey="Identificatie"/>
+                <Param name="Accept-Crs" value="EPSG:4326"/>
+                <Param name="expand" value="zaakinformatieobjecten" />
+                <Param name="Authorization"  sessionKey="originalMessage" xpathExpression="concat('Bearer ', $myToken)">
+                    <Param name="myToken" sessionKey="myToken" defaultValue="@@zaken-api.jwt@@"/>
+                </Param>
+                <Forward name="success" path="JsonToXml" />
+                <Forward name="exception" path="ErrorJsonToXml" />
+            </SenderPipe>
+            <JsonPipe name="ErrorJsonToXml">
+                <Forward name="success" path="buildErrorMsg" />
+            </JsonPipe>
+            <XsltPipe name="buildErrorMsg"
+                styleSheetName="Common/xsl/ParseNegativeHttpResult.xsl">
+                <Param name="senderPipeName" value="GetZgwZaakByIdentificatie" />
+                <Forward name="success" path="EXCEPTION" />
+            </XsltPipe>
+
+            <JsonPipe name="JsonToXml">
+                <Forward name="success" path="checkCount"/>
+            </JsonPipe>
+
+            <XmlIfPipe name="checkCount" xpathExpression="root/count = 0">
+                <Forward name="then" path="BuildErrorMessage" />
+                <Forward name="else" path="UnwrapOpenZaakApiEnvelopeToList" />
+            </XmlIfPipe>
+
+            <XsltPipe
+                name="BuildErrorMessage"
+                getInputFromFixedValue="&lt;dummy/&gt;"
+                styleSheetName="Common/xsl/BuildError.xsl"
+                storeResultInSessionKey="Error">
+                <Param name="cause" sessionKey="Error" type="DOMDOC" />
+                <Param name="code" value="TechnicalError" /> 
+                <Param name="reason" pattern="No zaak with the identificatie:{Identificatie} is found in GetZgwZaakByIdentificatie" />
+                <Param name="detailsXml" type="DOMDOC"/>
+                <Forward name="success" path="EXCEPTION" />
+                <Forward name="exception" path="EXCEPTION" />
+            </XsltPipe>
+
+            <XsltPipe
+                name="UnwrapOpenZaakApiEnvelopeToList"
+                styleSheetName="Common/xsl/UnwrapOpenZaakApiEnvelopeToList.xslt"
+                >
+                <Param name="Type" value="ZgwZaak"/>
+                <Param name="List" value="ZgwZaken"/>
+                <Forward name="success" path="AdjustExpandedElementNames"/>
+                <Forward name="error" path="EXCEPTION"/>
+            </XsltPipe>            
```

Comment:
This is already part of the getZgwZaak adapter that is also used for the non-expand version. Should just add an 'expand' param inside getZgwZaak adapter HttpSender as query param and set the attribute 'ignoreEmptyParameters' or something on the HttpSender. 'AdjustExpandedElementNames' can be added to the GetZgwZaak adapter too.

if api version < 1.5 then call GetZgwZaak adapter without expand+ GetZaakInformatieObjectenByZaak adapter otherwise call GetZgwZaak with expand='zaakinformatieobjecten'.

Then the iterator pipe needs to have the 'elementXPathExpression' changed to `//ZgwZaakInformatieObject`, so that it works for both the expand and the non-expand version.